cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/SingleBrokerStats.java

```diff
@@ -45,6 +54,15 @@ BasicStats basicStats() {
     return _basicStats;
   }
 
+  Map<String, Double> utilByDisk() {
+    return _utilByDisk;
+  }
+
+  double pctForDisk(String logdir) {
+    return _capacityByDisk.get(logdir) < 0 ? -1 :
```

Comment:
Should ensure that `_capacityByDisk` is never `null` in constructor?

Reply:
Change the logic in `Broker` to make `_utilByDisk` and `_capacityByDisk` never be null.